lib/partition_setup.pm

```diff
@@ -269,6 +269,7 @@ sub addlv {
 
     assert_screen 'expert-partitioner';
     send_key $cmd{system_view};
+    sleep(2);
```

Comment:
Instead of `sleep` (that in general is a solution that is never accepted because it solve problem for one case, but with different speed of the machines could not make it in future and what is worse, it creates a pattern for other developers to follow leaving the code as field of mine to debug) I'd like to suggest to try other thing, for example:
Given that the problem is that `alt-s` + `alt-home` got queued and `volume_management_feature` got asserted first, so the keys arrives later on and `send_key_until_needlematch('lvm_uncollapse_vgs', 'right')` does not know where the flow is, what we could try is create this repetition:
```
send_key_until_needlematch('volume_management_feature', 'down');
wait_still_screen(stilltime => 2, timeout => 4);
# Ensure Volume Management selected due to in slower archs sporadically root tree selection keys arrives with a delay
send_key_until_needlematch('volume_management_feature', 'down'); 
```

Reply:
I suspect it will fail at second iteration, but let's see.

Reply:
Ok, this works. I updated the VRs in description, all 10 passed.